ros2/src/airsim_ros_pkgs/src/airsim_ros_wrapper.cpp

```diff
@@ -1404,6 +1429,48 @@ std::shared_ptr<sensor_msgs::msg::CompressedImage> AirsimROSWrapper::get_jpeg_ms
     return jpeg_message;
 }
 
+std::shared_ptr<image_metadata::Metadata> AirsimROSWrapper::get_jpeg_msg_metadata()
+{
+    std::string vehicle_name = "Copter";
+    auto height_data = airsim_client_->getDistanceSensorData("Distance1", vehicle_name);
+    auto distance_data = airsim_client_->getDistanceSensorData("Distance2", vehicle_name);
+    auto gps_data = airsim_client_->getGpsData("Gps", vehicle_name);
+
+    auto vehicle_attitude = get_camera_attitude_euler_msg(vehicle_name, "front_center_fixed");
+    auto gimbal_attitude = get_camera_attitude_euler_msg(vehicle_name, gimbal_vehicle_reference_camera_name_);
+
+    auto metadata = std::make_shared<image_metadata::Metadata>(
+        image_metadata::Metadata{
+            distance_data.distance, // rangefinder distance
+            {vehicle_attitude->roll, vehicle_attitude->pitch, vehicle_attitude->yaw}, // drone attitude
+            {gps_data.gnss.geo_point.latitude, gps_data.gnss.geo_point.longitude,
+                gps_data.gnss.geo_point.altitude - origin_geo_point_.altitude}, // drone gps
+            3., // fix state
+            {gimbal_attitude->roll, gimbal_attitude->pitch, gimbal_attitude->yaw}, // gimbal attitude
+            7000, // gps_counter
+            240., // horizontal_accuracy
+            48., // horizontal_dilution_precision
+            "mission", // mission ID
+            7, // num_glonass_satellites_used
+            11, // num_gps_satellites_used
+            30, // num_total_satellites_used
+            97.0, // position_dilution_precision
+            height_data.distance, // relative_height
+            1, // rtk_connection_status
+            {gps_data.gnss.geo_point.latitude, gps_data.gnss.geo_point.longitude,
+                gps_data.gnss.geo_point.altitude - origin_geo_point_.altitude}, // rtk_position
+            40, // rtk_position_info
+            {-0.326, -9.269, 152.085}, // rtk_velocity

```

Comment:
assuming this is just dummy values for now?

Reply:
Yeah, I thought we decided that this doesn't need to be simulated.  Is that not correct?

Reply:
yeah, right. just wanted to make sure